app/snapshot.go

```diff
@@ -26,6 +26,7 @@ func SnapshotCmd() cli.Command {
 			SnapshotLsCmd(),
 			SnapshotRmCmd(),
 			SnapshotPurgeCmd(),
+			SnapshotPurgeStatusCommand(),
```

Comment:
The client needs to check if any replica is purging before proceeding as well.

By default, the purge command should return an error if a purge is in progress. We will need a flag `--skip-if-in-progress` for the purge command return success if purging is happening. The manager would need this flag for all the commands.